config/config.go

```diff
@@ -193,6 +193,7 @@ type GlobalConfig struct {
 	SlackURL        string `yaml:"slack_url"`
 	PagerdutyURL    string `yaml:"pagerduty_url"`
 	OpsGenieAPIHost string `yaml:"opsgenie_api_host"`
+	ExternalURL     string `yaml:"external_url"`
```

Comment:
That's another angle. I can see arguments for both sides.
Went back to the flag for now – should be ready then.